### ISSUE TITLE
Provide a Group content entity

### ISSUE THREAD

**Issue by amitaibu** [amitaibu/og#205]
OG already works with any fieldable entity.
We could take it one step further and provide a `Group` entity so site builders can use it instead of a node which is also a group.

**Comment by kristiaanvandeneynde** [comment]
This actually made me chuckle: I just got here from your tweet about OG8 and this is the first issue I see in your tracker 😃

But in all seriousness, this is the core idea behind the Group module. I'm definitely not going to ask you to rethink this, as there's an ever so slim chance someone will run OG and Group on the same website. But I do like to point out that the Group module uses that very same name for a content entity.

It's something to consider, I guess.

P.S.: Really curious to see OG8 in action!

**Comment by amitaibu** [comment]
Hi, happy to have you on OG issue queue :)

> But I do like to point out that the Group module uses that very same name for a content entity.

Sure, I have no problem to prefix the entity and have it called `OgGroup`.

> this is the core idea behind the Group module

Indeed, however in OG it's just a serving suggestion. In fact experience shows (at least in D7 days) that a group has to act also as content more often than not (publish/ unpublished; node access; etc')

**Comment by kristiaanvandeneynde** [comment]
> Sure, I have no problem to prefix the entity and have it called OgGroup.

Sweet, that will save us both head aches when someone does think combining OG and Group is a fun thing to do 😄 

> Indeed, however in OG it's just a serving suggestion. In fact experience shows (at least in D7 days) that a group has to act also as content more often than not (publish/ unpublished; node access; etc')

Yeah, I've noticed that as well. On the bright side, Deeson's given me time to work on core as an OKR and I've targeted this issue as what I want to work on: https://www.drupal.org/node/777578

I'm about 60% through that and once/if that lands, it would mean grants are available for all content entity types. That should open up a whole new world for access control modules like ours!

P.S.: Will you be in Milan for the dev days?
